packages/babel-traverse/src/path/family.js

```diff
@@ -203,9 +210,12 @@ export function getBindingIdentifierPaths(
     }
   }
 
+  // $FlowIssue Object.create() is object type
```

Comment:
Doesn't using `const id:  { [string]: NodePath } = Object.create(null);` above work?

Reply:
No flow says that the result of `Object.create()` is not compatible to object type. facebook/flow#4967